<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>波普特酒店温控系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../img.qrc">
    <normaloff>:/icon/images/xiaohui.png</normaloff>:/icon/images/xiaohui.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family:century gothic
}

#Form{
	background:url(:/picture/images/background.jpg);
}

QFrame
{
	background:#333;
	border-radius:15px;
}

QToolButton
{

background:red;
border-radius:60px;
}

QLabel
{
	color:white;
}

QPushButton
{
background:#B62B2B;
color:white;
border-radius:15px;
}

QPushButton:hover
{

color:red;
border-radius:15px;
background:#49ebff;
}

QLineEdit
{

background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}		

QComboBox{
background:white;
color:black;
border-radius:0px;
}

QComboBox QAbstractItemView {
border-radius:0px;
    outline: 0px solid gray;   /* 选定项的虚框 */
    color: white;
    background: #333;   /* 整个下拉窗体的背景色 */
     /* 整个下拉窗体被选中项的背景色 */
}
QStackedWidget
{

background-color:#333;
}		

QTableWidget::item
{
color:white;
background:#333;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>56</y>
     <width>684</width>
     <height>483</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_left">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton_report">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>113</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>13</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Report</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_checkinout">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>113</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>13</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Reception</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_return">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>430</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../home.png</normaloff>../home.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>5</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget
{
color:white;
border
}
border-bottom: 1px solid white;


QListWidget::item
{
border-bottom: 1px solid white;
color:white;

}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>546</width>
         <height>463</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#333;</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_checkinout">
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>1</x>
           <y>-5</y>
           <width>544</width>
           <height>467</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QFrame" name="frame_up">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QComboBox" name="comboBox_floor">
             <property name="geometry">
              <rect>
               <x>124</x>
               <y>24</y>
               <width>258</width>
               <height>34</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>century gothic</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border:1px solid white;
background:#333;
color:white;
border-radius:0px;


QComboBox::QAbstractItemView {
border-radius:0px;
    outline: 0px solid gray;   /* 选定项的虚框 */
    color: white;
    background: #333;   /* 整个下拉窗体的背景色 */
     /* 整个下拉窗体被选中项的背景色 */
}</string>
             </property>
             <item>
              <property name="text">
               <string>Click to select the floor</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1st</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2nd</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3rd</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4th</string>
              </property>
             </item>
            </widget>
            <widget class="QLabel" name="label_floor">
             <property name="geometry">
              <rect>
               <x>47</x>
               <y>24</y>
               <width>67</width>
               <height>36</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>century gothic</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Floor:</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_down">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>5</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QTableWidget::item {
    height: 60px;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="pushButton_printbill">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>330</y>
               <width>120</width>
               <height>40</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>century gothic</family>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background:#B62B2B</string>
             </property>
             <property name="text">
              <string>Export Bill</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_checkin">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>260</y>
               <width>120</width>
               <height>40</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>century gothic</family>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background:#B62B2B</string>
             </property>
             <property name="text">
              <string>Check In</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_printdetail">
             <property name="geometry">
              <rect>
               <x>320</x>
               <y>330</y>
               <width>120</width>
               <height>40</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>century gothic</family>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background:#B62B2B</string>
             </property>
             <property name="text">
              <string>Export Detail</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_checkout">
             <property name="geometry">
              <rect>
               <x>320</x>
               <y>260</y>
               <width>120</width>
               <height>40</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>century gothic</family>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background:#B62B2B</string>
             </property>
             <property name="text">
              <string>Check Out</string>
             </property>
            </widget>
            <widget class="QListWidget" name="listWidget_room">
             <property name="geometry">
              <rect>
               <x>17</x>
               <y>7</y>
               <width>81</width>
               <height>301</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>century gothic</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border:1px solid white;
border-radius:0px;

QListWidget {
border-radius:0px;
}


QListWidget::item
{
border-bottom: 1px solid white;
color:white;

}</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QTableWidget" name="tableWidget_status">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>10</y>
               <width>401</width>
               <height>241</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>century gothic</family>
               <pointsize>13</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">
QTableWidget {

gridline-color: white;
border-radius:0px;
}


QTableWidget::item
{
color:white;
background:#333;
}

QHeaderView::section {
    background:#333; /* 设置表头的背景颜色 */
    color: white;           /* 设置表头文本颜色 */
	border: 1px solid white;
 

}

QTableCornerButton::section {
    background-color: #333; /* 设置左上角表头的背景颜色 */
    color: white;           /* 设置左上角表头的文本颜色 */
}</string>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="lineWidth">
              <number>2</number>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="horizontalScrollMode">
              <enum>QAbstractItemView::ScrollPerPixel</enum>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>19</number>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>200</number>
             </attribute>
             <attribute name="verticalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>32</number>
             </attribute>
             <row>
              <property name="text">
               <string>ON/OFF</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Mode</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Current Temperature</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Target Temperature</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Wind Speed</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Total Cost</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <item row="0" column="0">
              <property name="text">
               <string/>
              </property>
             </item>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_report"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>-220</x>
     <y>-200</y>
     <width>1021</width>
     <height>821</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 60);</string>
   </property>
  </widget>
  <zorder>textBrowser_3</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources>
  <include location="../img.qrc"/>
 </resources>
 <connections/>
</ui>
